### PR TITLE
Add test /functional/tpm_policy-sanity-on-localhost

### PR DESCRIPTION
**Pull request by kkaarreell**

### CODE REVIEW

**Comment by kkaarreell** [comment]
Rawhide is still failing most likely due to bug https://bugzilla.redhat.com/show_bug.cgi?id=2008179

File: functional/tpm_policy-sanity-on-localhost/test.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        limeBackupConfig
+        # if IBM TPM emulator is present
```
Comment (sergio-correia) [inline functional/tpm_policy-sanity-on-localhost/test.sh:11]: This comment is outdated, no? I think now we check for `swtpm`?

**Review by sergio-correia — APPROVED** [review]
LGTM, test passes here with both swtpm and a qemu-backed tpm.

File: functional/tpm_policy-sanity-on-localhost/test.sh (hunk 1 above)
Comment (kkaarreell) [reply functional/tpm_policy-sanity-on-localhost/test.sh:11]: Right. I will remove it.. I guess there are more files with this comment.

**Pull request merged**